runtime/compiler/x/codegen/J9X86Instruction.cpp

```diff
@@ -71,14 +71,11 @@ TR::Snippet *TR::X86MemImmSnippetInstruction::getSnippetForGC()
 void TR::X86MemImmSnippetInstruction::assignRegisters(TR_RegisterKinds kindsToBeAssigned)
    {
    TR::X86MemImmInstruction::assignRegisters(kindsToBeAssigned);
-   if (kindsToBeAssigned & (TR_X87_Mask | TR_FPR_Mask))
+   if (kindsToBeAssigned & TR_FPR_Mask)
```

Comment:
Can you confirm if TR_X87_Mask is ever set? If its not, then this probably doesn't depend on eclipse-omr/omr#7799 and we can avoid a coordinated merge situation.